### PR TITLE
Fat - 188 setup mod notes tests

### PR DESCRIPTION
**Pull request by natali-zaitseva**
Setup tests for mod-notes module

### CODE REVIEW

**Review by psmagin — APPROVED** [review]
@natali-zaitseva you moved work on mod-notes karate test much forward! Great job!

**Comment by oleksandr-dekin** [comment]
Do we need empty file - mod-notes/src/main/resources/domain/mod-notes/features/note-links.feature ?

File: mod-notes/src/main/resources/karate-config.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    testTenant: testTenant ? testTenant: 'testTenant',
+    testAdmin: {tenant: testTenant, name: 'test-admin', password: 'admin'},
+    testUser: {tenant: testTenant, name: 'test-user', password: 'test'},
+
+    // define global features
+    login: karate.read('classpath:common/login.feature'),
+    dev: karate.read('classpath:common/dev.feature'),
+
+    // define global functions
+    uuid: function () {
```
Comment (oleksandr-dekin) [inline mod-notes/src/main/resources/karate-config.js:27]: Please, remove unused function in this module

**Comment by natali-zaitseva** [comment]
> Do we need empty file - mod-notes/src/main/resources/domain/mod-notes/features/note-links.feature ?

that is the placeholder for the future implementation. We can remove it for now

**Review by oleksandr-dekin — APPROVED** [review]

**Pull request merged**